diagnostics/ENSO_MSE/COMPOSITE/get_clima_in.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np

```

Comment:
You can probably combine most, or all, of the get_()_in.py modules into one read routine. Define a module, or modules, with separate routines to perform computations onto the data arrays after you read them in, and call them as needed from the composite driver routine.